bare-metal/apple-silicon/troubleshooting/cant-connect-using-ssh.mdx

```diff
@@ -0,0 +1,33 @@
+---
+meta:
+  title: Resolving SSH connection issues to Scaleway Mac mini
+  description: Learn how to fix SSH connection problems with Scaleway Apple silicon Mac mini by uploading your SSH key and rebooting the device.
+content:
+  h1: Resolving SSH connection issues to Scaleway Mac mini
+  paragraph: Learn how to fix SSH connection problems with Scaleway Apple silicon Mac mini by uploading your SSH key and rebooting the device.
+tags: ssh ssh-key connection
+dates:
+  validation: 2024-06-12
+  posted: 2024-06-12
+categories:
+  - bare-metal
+---
+
+This documentation addresses an issue faced by users who are unable to connect to their Mac Mini via SSH.
+This problem typically arises because the user's SSH key has not been uploaded to the Mac Mini.
```

Comment:
We may want to mention that the user could have blacklisted himself after failed VNC auth as well

```suggestion
This problem typically arises because the user's SSH key has not been uploaded to the Mac Mini.
It can also happen after multiple authentication failures while trying to access the VNC server because of default fail2ban security policies
```